arch/x86_64/kernel/gdt/gdt.asm

```diff
@@ -0,0 +1,17 @@
+bits 64
+
+global load_gdt
+load_gdt:
+    lgdt [rdi]
+    
+    push 0x08
+    push .flush
+    retfq
+    .flush:
+        mov ax, 0x10
+        mov ds, ax
+        mov es, ax
+        mov ss, ax
+        mov fs, ax
+        mov gs, ax
+        ret
```

Comment:
I thought we wouldn't have asm anymore

Reply:
You can't avoid assembly everywhere
It's just reloading segment registers..